kakao/src/main/kotlin/com/agoda/kakao/picker/KDatePicker.kt

```diff
@@ -0,0 +1,30 @@
+package com.agoda.kakao.picker
+
+import android.widget.DatePicker
+import androidx.test.espresso.contrib.PickerActions
+import com.agoda.kakao.common.views.KBaseView
+import com.agoda.kakao.text.KButton
+
+/**
+ * View for interact with default date picker
+ *
+ * @see DatePicker
+ */
+class KDatePicker : KBaseView<KDatePicker>({ isInstanceOf(DatePicker::class.java) }) {
```

Comment:
`DatePicker` is a default widget and you should leave the ability of matching it by Kakao users. Please provide all base class constructors without additional logic.

Reply:
We have the same logic for SnackBar

Reply:
Yeah, but you can't use snackbar view directly, only through the `Snackbar.show()`.
Whereas DatePicker and TimePicker are normal widgets and can be included in any layout.